en/05_Contributing/05_Coding_Conventions/01_PHP_Coding_Conventions.md

```diff
@@ -157,6 +157,16 @@ Order code in classes in the following order:
 - If you do write raw SQL, use double quotes around table/column names and use parameterised queries, for example `->where(['"Score" > ?' => 50])`.
 - Use [ANSI SQL](https://en.wikipedia.org/wiki/SQL#Standardization) format.
 
+## PHPDoc
+
+PHPDocs are not only useful when looking at the source code, but are also used in the API documentation at <https://api.silverstripe.org>.
+
+- All [public API](/project_governance/public_api) should have a PHPDoc to describe its purpose.
+- All `DataObject` and `Extension` subclasses should have `@method` annotations in their PHPDoc for relations (`has_one`, `has_many`, etc).
```

Comment:
```suggestion
- All `DataObject` and `Extension` subclasses must have `@method` annotations in their PHPDoc for relations (`has_one`, `has_many`, etc).
```
Since it's being enforced by CI it's a must

Reply:
The word "must" doesn't appear anywhere else in this document. Really it shouldn't matter whether these are enforce by CI or not. But I'll change this word since in this case yes, I suppose it would fail peer review (CI not-withstanding) if the annotation is missing.